clang/lib/CIR/CodeGen/CIRGenBuiltinAArch64.cpp

```diff
@@ -1607,6 +1599,50 @@ static mlir::Value buildArmLdrexNon128Intrinsic(unsigned int builtinID,
   }
 }
 
+mlir::Value buildAArch64NeonCall(unsigned int builtinID, CIRGenFunction &cgf,
+                                 llvm::SmallVector<mlir::Type> argTypes,
+                                 llvm::SmallVector<mlir::Value, 4> args,
+                                 llvm::StringRef intrinsicName,
+                                 mlir::Type funcResTy, mlir::Location loc,
+                                 bool isConstrainedFPIntrinsic = false,
+                                 unsigned shift = 0, bool rightshift = false) {
+  // TODO: Consider removing the following unreachable when we have
+  // metaDataTypes and buildConstrainedFPCall features implemented
+  assert(!MissingFeatures::metaDataTypes());
+  assert(!MissingFeatures::buildConstrainedFPCall());
```

Comment:
RFC: seeking for better approach, 

having missingfeature assert here to remind that turning those unreachables rely on these features.

having them again in following code so I can have skeleton similar to OG, and in the future easily find where to add code back in.

Reply:
Thanks for mapping these. We don't need the `metaDataTypes` because it's implied to be handled when `buildConstrainedFPCall` is implemented.